zqd/space/space.go

```diff
@@ -69,6 +72,34 @@ func (s Space) Name() string {
 	return filepath.Base(s.path)
 }
 
+func (s Space) Info() (api.SpaceInfo, error) {
```

Comment:
I've been trying to be disciplined about not bringing `api` into this package. `space`, `packet`, and `search` shouldn't be aware of protocol level stuff.

Reply:
I'm not seeing what this rule gets us given that the "protocol level stuff" in this instance is merely a struct populated with space fields and that is exclusively the space's concern.

But it's cosmetic and small, so adopted it in the interest of wrapping this PR up.